src/validation.cpp

```diff
@@ -590,6 +590,7 @@ class MemPoolAccept
         const CChainParams& m_chainparams;
         const int64_t m_accept_time;
         const bool m_bypass_limits;
+        const bool m_bypass_timelocks;
```

Comment:
I think I would get even further and have `m_bypass_absolute_timelocks`/`m_bypass_relative_timelocks`. 

You might be willingly to shrug about the relative timelocks encumbering the nSequence of a transaction while being sure the nLocktime make sense (e.g spotting bug in your anti-fee snipping logic).

Reply:
How common is it to have both in a transaction? 😮
If it's very common then sure. But also for this particular case, checking absolute but not relative, calling another `testmempoolaccept` without `bypass_timelocks` and checking that it fails on `non-BIP68-final` would do the trick, since nLockTime is checked before the nSequences.